packs/data/spell-effects.db/spell-effect-incendiary-aura.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "pE2TWhG97XbhgAdH",
+    "img": "systems/pf2e/icons/spells/incendiary-aura.webp",
+    "name": "Spell Effect: Incendiary Aura",
+    "system": {
+        "description": {
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Incendiary Aura]{Incendiary Aura}</em></p>\n<p>Each time a creature within the emanation takes fire damage, it catches on fire, taking 2d4 persistent fire damage.</p>\n<p>[[/r (ceil(@item.level/2)+1)d4[persistent,fire]]]{Leveled Persistent Fire Damage}</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "Aura",
+                "radius": "ceil(@item.level/2)*5+5",
+                "slug": "incendiary-aura",
+                "traits": []
```

Comment:
```suggestion
                "slug": "incendiary-aura",
                "traits": []
```
Blank traits?

Reply:
I wasn't sure if it should have the traits of the spell, which is "fire", "oracle", "cursebound". Happy to add them.

Reply:
Yeah, probably should.